src/Extractor/FileTextExtractor.php

```diff
@@ -1,12 +1,19 @@
 <?php
 
+namespace SilverStripe\TextExtraction\Extractor;
+
+use SilverStripe\Core\Config\Config,
+    SilverStripe\Core\Injector\Injector,
+    SilverStripe\Core\ClassInfo;
```

Comment:
This style of importing namespaces isn't PSR-2 compliant, but the codesniffer will fix it for you automatically 👍

Reply:
> There MUST be one use keyword per declaration.

Well I'll be darned (I hate it, but thanks!), I'll leave it to be auto-fixed.